_sass/_objects--notify-form.scss

```diff
@@ -0,0 +1,18 @@
+$size--input-height:                          45px;
+$size--input-width:                           305px;
+$color--input-border:                         $alto;
+
+.notify-form {
+  &-input {
+    display: inline-block;
+    height: $size--input-height;
+    padding: 0 $input-padding-x;
```

Comment:
So you saw the variable, maybe the doubt is if you should set the whole input style for all the app, but I will say yes, your input forms will look like the one within the contact section